warplib/integer.cairo

```diff
@@ -0,0 +1,15 @@
+use integer::u128_try_from_felt;
```

Comment:
All logic related with conversions are in `warplib/math/utils.cairo` I would put these functions there

Reply:
Ok, then I will translate that file to cairo1 so we can work with it